CONTRIBUTING.md

```diff
@@ -1,17 +1,17 @@
 ## Contributing Guidelines
-This repo contains an alpha release of some of the foundational aspects of V2 of Kubernetes Federation. All the contribution is done via GitHub. Please also read the contributors guide: https://github.com/kubernetes/community/blob/master/contributors/guide/README.md
+This repo contains an alpha release of some of the foundational aspects of Kubernetes Cluster Federation. All the contribution is done via GitHub. Please also read the contributors guide: https://github.com/kubernetes/community/blob/master/contributors/guide/README.md
 
 ## Sign the CLA
 Kubernetes projects require that you sign a Contributor License Agreement (CLA) before we can accept your pull requests. Please see https://github.com/kubernetes/community/blob/master/CLA.md for more information
 
 ## Learn About Kubernetes Cluster Federation
-If you want to learn about Kubernetes Cluster Federation, please read through the doc:
-https://github.com/kubernetes-sigs/kubefed#kubernetes-cluster-federation
+If you want to learn about KubeFed, please read through the doc:
```

Comment:
Please revert as per a previous comment - any reference to 'Kuberentes Cluster Federation' should remain unchanged.

Reply:
Yeah, I left this one on purpose. I thought it was nice to have the full name, then the short name, to introduce both. Either way is pretty good I think.

Reply:
Fine to introduce the name, but changing the readme link resulted in a broken link.

Reply:
Oh I see, right. Good catch. :)